projects/core/src/lib/components/feature-info/feature-info/feature-info.component.ts

```diff
@@ -84,4 +103,40 @@ export class FeatureInfoComponent implements OnInit, OnDestroy {
       });
   }
 
+  private featureInfo3DToResponse(cesiumFeatureInfo: FeatureInfo3DModel): FeatureInfoResponseModel {
+    let layerId: string | null = null;
+    this.mapService.getCesiumLayerManager$().pipe(take(1)).subscribe(cesiumLayerManager => {
```

Comment:
As metioned before, RxJS should not be used to execute synchronous code like this.